deploy.rb

```diff
@@ -6,6 +6,12 @@ def on_standby?
   current_hostname == standby_hostname
 end
 
+dep 'webpack compile during deploy', :env, :deploying, template: 'task' do
+  requires 'npm packages installed'
```

Comment:
This dep lives in `sharejs.rb`. All the deps in this repo are in a global namespace so in most ways it doesn't really matter, but maybe it makes sense to move that dep into this file if it's going to be used across multiple projects?